Build/GameSettings.h

```diff
@@ -73,11 +73,10 @@ struct GAME_SETTINGS
 //Enums for the difficulty levels
 enum
 {
-	DIF_LEVEL_ZERO,
-	DIF_LEVEL_EASY,
-	DIF_LEVEL_MEDIUM,
-	DIF_LEVEL_HARD,
-	DIF_LEVEL_FOUR,
+	DIF_LEVEL_EASY = 1,
```

Comment:
What I meant with zero/one-based difficulty (not enum) is that 'easy' equals 0 with GIO_ enum and 1 with DIF_ enum. I've decided to keep it that way with DIF_ enum to ensure backward compatibility. I removed DIF_LEVEL_ZERO (wasn't used anywhere) and started numbering DIF_ enum with with 1 instead.